superset-frontend/src/SqlLab/components/HighlightedSql/HighlightedSql.test.tsx

```diff
@@ -17,12 +17,9 @@
  * under the License.
  */
 import React from 'react';
-import SyntaxHighlighter from 'react-syntax-highlighter';
-import { mount, shallow } from 'enzyme';
 
 import HighlightedSql from 'src/SqlLab/components/HighlightedSql';
-import ModalTrigger from 'src/components/ModalTrigger';
-import { supersetTheme, ThemeProvider } from '@superset-ui/core';
+import { fireEvent, render } from 'spec/helpers/testing-library';
```

Comment:
Thank you @justinpark for the improvements! We generally avoid `describe` and `it` in RTL tests. Could you also update those?